apps/src/dance/Dance.js

```diff
@@ -691,3 +693,36 @@ Dance.prototype.captureThumbnailImage = function () {
     setThumbnailBlobFromCanvas(canvas);
   }
 };
+
+Dance.prototype.doAi = async function (input) {
```

Comment:
I am unsure if this is temporary code that will be replaced. But perhaps we could rename this function to something like `postOpenaiRequest`?

Reply:
As discussed in https://github.com/code-dot-org/dance-party/pull/650, let's improve the naming as our design gets clearer in the next round of work, which should happen within a week or so.